src/Joomlatools/Console/Command/Site/Listing.php

```diff
@@ -16,15 +16,89 @@ class Listing extends Database\AbstractDatabase
     {
-        parent::configure();
-
         $this
             ->setName('site:list')
-            ->setDescription('List add Joomla site');
+            ->setDescription('List Joomla sites')
+            ->setHelp('List Joomla sites running on this box');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        parent::execute($input, $output);
+        define('_JEXEC', true);
+        define('JPATH_BASE', true);
+        define('JPATH_PLATFORM', true);
+
+        $dir = new \DirectoryIterator('/var/www');
+        $sites = array();
+
+        $canonical = function($version) {
+            if (isset($version->RELEASE)) {
+                return 'v' . $version->RELEASE . '.' . $version->DEV_LEVEL;
+            }
+
+            // Joomla 3.5 and up uses constants instead of properties in JVersion
+            $className = get_class($version);
+            if (defined("$className::RELEASE")) {
+                return 'v'. $version::RELEASE . '.' . $version::DEV_LEVEL;
+            }
+
+            return 'unknown';
+        };
+
+        foreach ($dir as $fileinfo)
+        {
+            $code = $application = null;
+
+            if ($fileinfo->isDir() && !$fileinfo->isDot())
+            {
+                $files = array(
+                    'joomla-cms'           => $fileinfo->getPathname() . '/libraries/cms/version/version.php',
+                    'joomlatools-platform' => $fileinfo->getPathname() . '/lib/libraries/cms/version/version.php',
+                    'joomla-1.5'           => $fileinfo->getPathname() . '/libraries/joomla/version.php'
+                );
+
+                foreach ($files as $type => $file)
+                {
+                    if (file_exists($file))
+                    {
+                        $code        = $file;
+                        $application = $type;
+
+                        break;
+                    }
+                }
+
+                if (!is_null($code) && file_exists($code))
+                {
+                    $identifier = uniqid();
+
+                    $source = file_get_contents($code);
+                    $source = preg_replace('/<\?php/', '', $source, 1);
+                    $source = preg_replace('/class JVersion/i', 'class JVersion' . $identifier, $source);
+
+                    eval($source);
+
+                    $class   = 'JVersion'.$identifier;
+                    $version = new $class();
+
+                    $sites[] = (object) array(
+                        'name'    => $fileinfo->getFilename(),
+                        'docroot' => $fileinfo->getFilename() . '/' . ($application == 'joomlatools-platform' ? 'web' : ''),
+                        'type'    => $application,
+                        'version' => $canonical($version)
+                    );
+                }
+            }
+        }
+
+        $output->write("\n");
+        $output->writeln("<comment>Sites Running On This Box:</comment>");
```

Comment:
@rodolfovillaruz I would remove this line: if you run the command you know what the result will be. It will also make it easier to parse if the output only contains the actual sites?